adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -275,7 +275,17 @@ private AuthenticationResult tryAcquireTokenSilent(final AuthenticationRequest a
 
         if (shouldTrySilentFlow(authenticationRequest)) {
             Logger.v(TAG, "Try to acquire token silently, return valid AT or use RT in the cache.");
-            authenticationResult = acquireTokenSilentFlow(authenticationRequest);
+            try {
```

Comment:
Is it OK to commit this branch into the release branch? Or merge the release branch here and keep committing into dev? @weijjia